bser.c

```diff
@@ -48,7 +50,8 @@ static int bser_real(double val, json_dump_callback_t dump, void *data)
   return dump((char*)&val, sizeof(val), data);
 }
 
-bool bunser_string(const char *buf, json_int_t avail, json_int_t *needed,
+/* Common codepath for bytestring and UTF-8 encoded strings. */
+static bool bunser_string(const char *buf, json_int_t avail, json_int_t *needed,
```

Comment:
This is actually fine -- in d44395242985339e85112bede3c02bca8c63b04b @dhruvsinghal renamed `bunser_string` to `bunser_bytestring`.